src/webots/nodes/utils/WbPaintTexture.hpp

```diff
@@ -39,7 +39,7 @@ class WbPaintTexture : public QObject {
 
   static bool isPaintable(const WbShape *shape);
 
-  void pickColor(WbRgb &pickedColor, const WbVector2 &uv) const;
+  void pickColor(WbRgb &pickedColor, const WbVector2 &uv, float *pickedDensity = NULL) const;
```

Comment:
It is bad to mix input and output arguments, we should also change the function so that first the input arguments are given and then the output arguments:
```suggestion
  void pickColor(const WbVector2 &uv, WbRgb &pickedColor, float *pickedDensity = NULL) const;
```

Reply:
I was hoping to avoid changing the other parts of the code. I will revert to `pickDensity()` as changing the order will affect other parts of the code and cause inconsistency. For example `WbAppearance::pickColorInTexture(WbRgb &pickedColor, const WbVector2 &uv)` will not be consistent with `WbPaintTexture::pickColor(const WbVector2 &uv, WbRgb &pickedColor, float *pickedDensity = NULL)` (argument order is reversed).

Reply:
I think that we should consider reversing the arguments in all the functions.
It is standard to have first input and then output arguments.